### DISCUSSION TITLE
Keyboard issues in desktop connections

### DISCUSSION THREAD

**Discussion by HJaegle** [gravitational/teleport#31073] [Q&A]
Hello community,
I'm new at teleport and i have two issues with desktop connections:
1. the numbers block on keyboard doesn't work within a teleport desktop connection. However it work in a regular RDP connection.
2. sometimes the keyboard doesn't work at all. In this case the workaround is to hit Cntr + Alt + End

Does anyone know this issue and knows a solution for it.

Thank you

**Comment by zmb3** [comment]
Are you using a keyboard layout that is not `en-US`?

**Reply by HJaegle** [reply]
Hey zmb3,
thank you for reply!
No we are using german (de-DE) keyboard layout.

**Reply by HJaegle** [reply]
Some issue with numlock when i using en-US keboard layout :(

**Comment by danielnoblia** [comment]
I have the same problem with the Swedish QVERTY keyboard layout. Sometimes it helps Alt + right-shift sometimes Win + right-shift. It seems to happen when switching to other tasks and coming back to the RDP Web session

**Comment by pveredas** [comment]
Hi,
Same here, using keyboard PT-PT and have the same issues mention above and one more. The keys does not respect the portuguese layout.
